source/guides/clear/mixer.rst

```diff
@@ -258,22 +288,18 @@ set to get a smaller kernel image, which will also be faster to load.
 
       $EDITOR ./local-bundles/editors
 
-#. List the bundles in the mix again to confirm removal.
+#. List the bundles in the mix again to confirm removal of :command:`joe`.
 
    .. code-block:: bash
 
       mixer bundle list  --tree
 
-
 #. Build bundles:
 
    .. code-block:: bash
 
       mixer build bundles
 
-   Look in ~/mixer/update/image/<mix version>/full for the full chroot after the
-   :command:`build` command completes.
-
 #. Build update content. Browse to your \http://localhost site and you'll see
```

Comment:
Please add a new line after "Build update content."  

Please add, "First, browse to your..."